Source/Tests/ELFCodeLoader2.h

```diff
@@ -220,6 +221,18 @@ class ELFCodeLoader2 final : public FEXCore::CodeLoader {
 
     bool LoadedWithFD = false;
     int FD = getauxval(AT_EXECFD);
+    char *FDString = getenv("FEX_EXECVEFD");
+
+    if (FDString) {
+      // If we passed the execve FD to us then use that.
+      char *EndPtr{};
+      FD = ::strtol(FDString, &EndPtr, 10);
+      if (EndPtr == FDString) {
+        LogMan::Msg::AFmt("FEXInterpreter passed invalid FD to exececute: {}", FDString);
+        return;
+      }
+      unsetenv("FEX_EXECVEFD");
```

Comment:
We pass envp to the guest application below.
We need to remove `FEX_EXECVEFD` getting passed to the guest down below on line 294.
Either doing string compares or using `environ` instead of `envp`

Reply:
I was mistaken here, documentation is a but unclear but unsetenv modifies envp in place. Resolved.